framework/godellauncher/param.go

```diff
@@ -38,7 +38,9 @@ type PluginsParam struct {
 
 type SinglePluginParam struct {
 	artifactresolver.LocatorWithResolverParam
-	Assets []artifactresolver.LocatorWithResolverParam
+	Assets           []artifactresolver.LocatorWithResolverParam
+	Override         bool
+	FromPluginConfig bool
```

Comment:
Looking at the code, I don't see a case in which this won't be true (it appears that every instance of constructing a `SinglePluginParam` comes from config). What's the intent behind the variable?

Reply:
Maybe I misunderstood how it was being used. I only set it to try in:

```
func (c *SinglePluginConfig) ToParam() (godellauncher.SinglePluginParam, error) {
```
The idea was it would be true if the `SinglePluginParam` came from a `SinglePluginConfig` but default to false in all other cases.

The reason that we need this is because (per the ticket) we want to error if there are 2 identical plugins found in the "plugins" portion of configuration. So we need to know if they came from the `plugin` configuration of the `configprovider`